### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by yashovardhan**
Added a new base documentation for Web3Auth. Please let me know if we need anything else in this base readme. I will add more READMEs later for each of our packages as well.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,59 +1,113 @@
...
+To understand what they are and what they're for: Check out [Adapters](https://docs.web3auth.io/api-reference/web/adapters/)
+
+Currently we have the following adapters available for utilisation:
+
+- [`@web3auth/openlogin-adapter`](https://npmjs.com/package/@web3auth/openlogin-adapter)
+- [`@web3auth/phantom-adapter`](https://npmjs.com/package/@web3auth/phantom-adapter)
+- [`@web3auth/torus-evm-adapter`](https://npmjs.com/package/@web3auth/torus-evm-adapter)
+- [`@web3auth/torus-solana-adapter`](https://npmjs.com/package/@web3auth/torus-solana-adapter)
+- [`@web3auth/metamask-adapter`](https://npmjs.com/package/@web3auth/metamask-adapter)
+- [`@web3auth/wallet-connect-v1-adapter`](https://npmjs.com/package/@web3auth/wallet-connect-v1-adapter)
```
Comment (himanshuchawla009) [inline README.md:102]: add coinbase adapter as well.

Comment (himanshuchawla009) [inline README.md:102]: add coinbase adapter as well. Also add some info about plugins may be now or later in readme.

**Review by himanshuchawla009 — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (himanshuchawla009) [reply README.md:102]: merging it for now. But plz make these updates in future pr. @yashovardhan

**Review by himanshuchawla009 — APPROVED** [review]
lgtm

**Review by chaitanyapotti — APPROVED** [review]
lgtm

**Pull request merged**